docs/clientdev/context.rst

```diff
@@ -1,10 +1,10 @@
 Context Management
 ==================
 
-All clients should have the concept of a concurrency safe context storage.
+All SDKs should have the concept of a concurrency safe context storage.
 What this means heavily depends on the language.  The basic idea is that a
 user of the Raven library can call a method to provide additional context
-information from whererever the user has access to the client object in a
+information from whererever the user has access to the SDK object in a
```

Comment:
This seems like a spot where the notion of "library is SDK, instance/class is client" could apply, but not sure how/where to differentiate that